blocklylib-core/src/main/java/com/google/blockly/android/ui/VirtualWorkspaceView.java

```diff
@@ -91,6 +92,14 @@ public VirtualWorkspaceView(Context context, AttributeSet attrs) {
 
     public VirtualWorkspaceView(Context context, AttributeSet attrs, int defStyleAttr) {
         super(context, attrs, defStyleAttr);
+        TypedArray a = context.getTheme().obtainStyledAttributes(attrs,
+                R.styleable.VirtualWorkspaceView, 0, 0);
+        try {
+            mDrawGrid = a.getBoolean(R.styleable.VirtualWorkspaceView_drawGrid,
```

Comment:
Can you move this up into WorkspaceFragment.onInflate (take a look at FlyoutFragment for an example), cache the value, and then call mVirtualWorkspaceView.setDrawGrid in onCreateView with the cached attribute?